src/pages/videoDetail/VideoDetail.jsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import { AiFillDislike, AiFillLike } from "react-icons/ai";
+import { BiTimeFive } from "react-icons/bi";
+import { CgPlayList } from "react-icons/cg";
+import { Link, useParams } from "react-router-dom";
+import { ThumbnailCard } from "../../components/index-component";
+import { useDataContext } from "../../context/data-context";
+
+export const VideoDetail = () => {
+  const param = useParams();
+  const { dataState } = useDataContext();
+  const videoID = param.undefined;
```

Comment:
suggestion : do const { videoID }= param;

Reply:
will work on this later